manifests/install.pp

```diff
@@ -12,12 +12,31 @@
   $proxy_host = $shibboleth_idp::proxy_host
   $proxy_port = $shibboleth_idp::proxy_port
 
+  if $shibboleth_idp::manage_user {
+    ensure_resource('user', $shibboleth_idp::shib_user, {
+        managehome => true,
+        system     => true,
+        gid        => $shibboleth_idp::shib_group,
+        shell      => '/sbin/nologin',
+    })
+
+    ensure_resource('group', $shibboleth_idp::shib_group, {
+        ensure => present
+    })
+  }
+
   file { $shibboleth_idp::shib_src_dir:
     ensure => directory,
     owner  => $shibboleth_idp::shib_user,
     group  => $shibboleth_idp::shib_group,
   }
 
+  file { $shibboleth_idp::idp_log_dir:
+    ensure => directory,
+    owner  => $shibboleth_idp::shib_user,
+    group  => $shibboleth_idp::shib_group,
+  }
```

Comment:
We do care, but the defaults were sufficient. I've added explicit mode values to clear, though.